resource/css/styles.css

```diff
@@ -1017,7 +1035,8 @@ th, td {
 }
 
 tr {
-  border-image: '../pics/stripe.png' 20 20 repeat;
+  display: flex;
```

Comment:
This causes layout problems on the vocabulary front page:

![skosmos-css-problems](https://user-images.githubusercontent.com/1132830/37520172-b295885c-2924-11e8-93d1-9830c7a361d6.png)

Are you sure it's a good idea to change all tables to flex layout? Will it work with older browsers?